### ISSUE TITLE
Shrink the repository by removing ancient big file.

### ISSUE THREAD

**Issue by eggpi** [eggpi/citationhunt#99]
In the very early commits of this repository, there used to be a database dump that takes a few tens of MB. I kind of forgot about it, but I've heard a few complaints of people trying to clone it, since it's about 120MiB to download.

Using the BFG repo-cleaner (https://rtyley.github.io/bfg-repo-cleaner/) would be a good way to rewrite history and remove that file. On my local repository, that brings the size down to 1MiB (btw, `git count-objects -vH` seems to be the command for measuring sizes).

Of course rewriting history and force-pushing is not a great idea, but it's not a terrible thing either for a repository with few forks like this, so I'm inclined to do it. @Nikerabbit @translatewiki would this potentially break your automation?

**Comment by Nikerabbit** [comment]
If `git fetch -q && git reset -q --hard origin/master` gets over it, then we don't need to do anything manually.

**Comment by eggpi** [comment]
Cool, thanks! So I just saw that the docs for BFG [recommend a fresh clone](https://rtyley.github.io/bfg-repo-cleaner/#usage), but I experimented a little by making a test clone of this repository then pulling/pushing to my local (already shrunk) copy, and it looks like `fetch && reset` will be fine.

The test clone didn't shrink by just doing `fetch && reset`, but that's probably not a problem for you. And importantly, pushing from the test clone to the shrunk one doesn't seem to reintroduce the file.

To get the test clone to shrink, I had to do `git reflog expire --expire=now --all && git gc --prune=now --aggressive`, as per BFG's docs, but I suspect git might just take care of it eventually. I think I'll go ahead with this later today then.
